app/views/content/service_guidance_provider.md

```diff
@@ -232,7 +232,7 @@ Candidates will also be able to use UCAS’s Apply 2 service for this part of th
 
 ### Informing candidates your courses are on Apply
 
-Your courses will automatically be uploaded to [Apply for teacher training](https://www.apply-for-teacher-training.service.gov.uk) as part of our onboarding process.
+Your courses will automatically be uploaded to [Apply for teacher training](https://www.gov.uk/apply-for-teacher-training) as part of our onboarding process.
```

Comment:
Similar to #3851 this link should redirect to the `candidate_interface_root_path` which based on the environment will either redirect to the environment start page or if in production to `GOVUK_APPLY_START_PAGE_UR`L which resolves to https://www.gov.uk/apply-for-teacher-training

Reply:
Do you have any advice on how to embed a ruby path in a markdown file? Because I'm not sure how it's done. Why is this file a md in the first place/ should it be converted to a html.erb?

Reply:
We use markdown for content files that don't require any elaborate formatting as they are easier to create and manage.
The conversion to html is handled by the [ContentHelper](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/helpers/content_helper.rb).

Reply:
It’s now possible to use ERB in Markdown. Wonder if this issue might have been addressed by #4018 and #4009?